addons/mail/static/src/models/message/message.js

```diff
@@ -442,6 +442,23 @@ function factory(dependencies) {
                 this.originThread.isModeratedByCurrentPartner
             );
         }
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsSubjectEqualThreadName() {
+            if (!this.subject || !this.originThread || !this.originThread.name) {
+                return false;
+            }
+            const prefix = ['re:', 'fw:', 'fwd:'];
+            let cleanedSubject = this.subject.toLowerCase();
+            prefix.forEach((pref) => {
```

Comment:
prefer for...of loop, they are easier to debug step by step because they don't require an intermediate callback